### DISCUSSION TITLE
Can the emmisive textures be applied to the minecart?

### DISCUSSION THREAD

**Discussion by Neuron67** [MoreMcmeta/core#49] [Help Wanted] [answered]
I know the description states that entities, block entities are compatible but most of them have always been except for the minecart so I just need to know if this More McMeta Emissive Textures Plugin can make it possible. Thanks

**Comment by soir20** [comment, marked as answer]
Yes, you can apply emissive overlays to minecarts on both Fabric and Forge.

![2023-11-05_00 31 47](https://github.com/MoreMcmeta/core/assets/71418127/9dde9cce-8ed2-4daa-b890-fc658aa6fea2)
